zcash_test_vectors/zip_0244.py

```diff
@@ -372,18 +371,18 @@ def randbytes(l):
         other_sighashes = {
             nHashType: None if txin is None else signature_digest(tx, t_inputs, nHashType, txin)
             for nHashType in ([
+                                  SIGHASH_ALL,
+                                  SIGHASH_NONE,
+                                  SIGHASH_SINGLE,
+                                  SIGHASH_ALL | SIGHASH_ANYONECANPAY,
+                                  SIGHASH_NONE | SIGHASH_ANYONECANPAY,
+                                  SIGHASH_SINGLE | SIGHASH_ANYONECANPAY,
+                                  ] if txin is None or txin.nIn < len(tx.vout) else [
```

Comment:
can you explain these changes?

Reply:
![image](https://github.com/user-attachments/assets/5214b26d-8bac-497d-aa02-d9e8aacdab4c)
Interesting, locally it doesn't mark this as a change for me. I'll look into it.

Reply:
Again, reverted the changes, they were unnecessary.